pkg/render/render_test.go

```diff
@@ -267,6 +396,10 @@ func componentCount(components []render.Component) int {
 	for _, c := range components {
 		objsToCreate, _ := c.Objects()
 		count += len(objsToCreate)
+		glog.Printf("Component: %s\n", reflect.TypeOf(c))
```

Comment:
Makes it easier to debug these tests when they fail - prints out each component and the resources emitted.